### PR TITLE
qemu_guest_agent: Add paramters to 'qemu_guest_agent_hotplug.cfg'

### PR DESCRIPTION
**Pull request by 6-dehan**
hot fix for this last file.

ID: 2029676
Signed-off-by: demeng <demeng@redhat.com>

### CODE REVIEW

**Comment by 6-dehan** [comment]
@vivianQizhu please help to review this. thanks
(05/10) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.qemu_guest_agent_hotplug.q35:PASS (210.61 s)
 (10/10) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_guest_agent_hotplug.q35: PASS (224.96 s)

File: qemu/tests/cfg/qemu_guest_agent_hotplug.cfg [hunk 1]
```diff
@@ -56,7 +56,12 @@
         gagent_stop_cmd = "net stop qemu-ga"
         gagent_status_cmd = sc query qemu-ga |findstr "RUNNING"
         gagent_pkg_check_cmd = wmic product get name |findstr "QEMU guest agent"
-        gagent_host_path = "/tmp/${qemu_ga_pkg}"
+        gagent_host_path = "/var/tmp/"
+        gagent_download_cmd = "wget https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/latest-qemu-ga/${qemu_ga_pkg} -O ${gagent_host_path}"
```
Comment (vivianQizhu) [inline qemu/tests/cfg/qemu_guest_agent_hotplug.cfg:60]: Need to remove Line 52?

**Comment by 6-dehan** [comment]
@vivianQizhu this could be reviewed

File: qemu/tests/cfg/qemu_guest_agent_hotplug.cfg (hunk 1 above)
Comment (vivianQizhu) [reply qemu/tests/cfg/qemu_guest_agent_hotplug.cfg:60]: Seems Line 52 still there, it is redundant with this line.

**Comment by 6-dehan** [comment]
what about now..

File: qemu/tests/cfg/qemu_guest_agent_hotplug.cfg (hunk 1 above)
Comment (6-dehan) [reply qemu/tests/cfg/qemu_guest_agent_hotplug.cfg:59]: > Seems Line 52 still there, it is redundant with this line.
sorry, maybe I force checkbranch to master and abort this part. 
what about now.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**